tables_io/io_layer.py

```diff
@@ -358,3 +461,187 @@ def readPqToDataframes(basepath, keys=None, **kwargs):
         except Exception:  #pragma: no cover
             pass
     return dataframes
+
+
+def readPqToArrayDict(infile):
+    """ Open a parquet file and return a dictionary of `numpy.array`
+    """
+    df = readPqToDataframe(infile)
```

Comment:
Is there a particular reason to go via a dataframe here, or just convenience?

Also this and the ones below could do with ``Parameters`` and ``Returns`` in the docstring if we want sphinx or similar to pick them up.  Unless they are internal?

Reply:
1. I was not planning on having internal methods.  If we see a use for it, someone else probably will to.  
2. I added Added Parameters and Returns to the docstring
3. Yes, it was just convenience.  I changed it to: 
```
    tab = pq.read_table(infile, columns=columns)
    return OrderedDict([(c_name, col.to_numpy()) for c_name, col in zip(tab.column_names, tab.itercolumns())])

```